google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/TimePartitioning.java

```diff
@@ -37,16 +37,20 @@
   private static final long serialVersionUID = -8565064035346940951L;
 
   /**
-   * The supported types are DAY, which will generate one partition per day, and HOUR, which will
-   * generate one partition per hour. (Providing an empty string used to cause an error, but in
-   * OnePlatform the field will be treated as unset.)
+   * The supported types are DAY, HOUR, MONTH, and YEAR which will generate one partition per time
+   * unit type. (Providing an empty string used to cause an error, but in OnePlatform the field will
+   * be treated as unset.)
    */
   public enum Type {
 
     /** Table is partitioned per day, based on data loading time. */
```

Comment:
the "based on data loading time" clause is only true for cases where timepartitioning is not based on user data columns.  Remove it from the various enum comments here.

Reply:
done